app/services/csv_exporter.rb

```diff
@@ -28,14 +29,16 @@ class CSVExporter
       'Third party',
       'Reply method',
       'SAR Subject type',
-      'SAR Subject full name'
+      'SAR Subject full name',
+      'Business unit responsible for late response'
   ]
 
   def initialize(kase)
     @kase = kase
   end
 
   #rubocop:disable Metrics/CyclomaticComplexity
+  #rubocop:disable Metrics/MethodLength
```

Comment:
could we put these rubocop:disable methods on the to_csv method
`def to_csv #rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength`
That way we don't have to disable and enable them